x-pack/plugins/observability/public/config/paths.ts

```diff
@@ -16,6 +19,8 @@ export const paths = {
       ruleId ? `${RULES_PAGE_LINK}/${encodeURI(ruleId)}` : RULES_PAGE_LINK,
     alertDetails: (alertId?: string | null) =>
       alertId ? `${ALERT_PAGE_LINK}/${encodeURI(alertId)}` : ALERT_PAGE_LINK,
+    cases: CASES_PAGE_LINK,
```

Comment:
Where do we use `cases` and `casesCreate`?

Reply:
@CoenWarmer, No need to declare these links. To handle the `Actions` you will need to use the cases UI hooks e.g. `casesUi.hooks.getUseCasesAddToExistingCaseModal`